inst/testdata/cdmModulesAnalysisSpecifications.json

```diff
@@ -565,8 +565,12 @@
             "type": "exit"
           }
         ],
-        "includeTreatments": "startDate",
-        "indexDateOffset": 0,
```

Comment:
Removing these parameters is a breaking change - should this have resulted in TreatmentPatterns v4.0 vs v3.1?

Reply:
No worries - just wanted to verify that. I'm aiming to do a patch release of Strategus and then will work to merge your work into `develop`. Thanks!